wasm_utils/src/api_serialization/verify_signature.rs

```diff
@@ -0,0 +1,8 @@
+use holochain_core_types::{agent::Base32, error::HolochainError, json::*};
+
+#[derive(Deserialize, Default, Clone, PartialEq, Eq, Hash, Debug, Serialize, DefaultJson)]
+pub struct VerifySignatureArgs {
```

Comment:
Lets make the arguments have a `Provenance` and a payload, not the pub_key and the signature.  I think this is semantically more accurate, and also, that way when we merge `crypto-conductor-1` then the verify function will just be super easy: `provenance.verify(payload)`  Because that's all ready [been added there](https://github.com/holochain/holochain-rust/blob/912b56dbae93b1a49b0c7f00cebd690e1afcf174/dpki/src/utils.rs#L17).

Reply:
Good idea. I would like to get the other PR merged before though. I would rather do this as a follow-up